### PR TITLE
Add systemd lsof procps dependencies to rust buster

### PR DESCRIPTION
**Pull request by jorgebay**
Add `systemd`, `lsof` and `procps` (for `ps`) to rust dev image.

### CODE REVIEW

**Review by c-nixon — APPROVED** [review]

File: rust/buster/Dockerfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ RUN curl https://sh.rustup.rs -sSf | \
     sh -s -- -y --default-toolchain $VERSION --component clippy rustfmt --profile minimal --no-modify-path \
     && chmod -R a+rw $RUSTUP_HOME $CARGO_HOME
 
-RUN cargo install cargo-audit
+RUN cargo install cargo-audit --version ~0.11
```
Comment (jorgebay) [inline rust/buster/Dockerfile:28]: I pinned to version 0.11 that is the last version that supports rust 1.42.

**Comment by jorgebay** [comment]
@c-nixon after its merged, should I push the image using `VERSION=1.42.0` ?

**Comment by c-nixon** [comment]
Hmm, I think it's probably time to bump to latest stable

**Comment by jorgebay** [comment]
ok, I'll pin cargo-audit to latest version and bump the rust version in Jenkinsfile.

**Review by c-nixon — APPROVED** [review]
LGTM

**Pull request merged**